src/components/general/AddStakeholderModal/SearchStakeholder/Stakeholder/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from 'react';
+import Avatar from '#dui/Avatar';
+import HighlightableTextOutput from '#components/viewer/HighlightableTextOutput';
+import {
+    ElementFragments,
+    DraggableContent,
+} from '@the-deep/deep-ui';
+
+import { Organization } from '#typings';
+
+import styles from './styles.scss';
+
+interface Props {
+    searchValue?: string;
+    value: Organization;
+}
+
+function Stakeholder(props: Props) {
+    const {
+        value,
+        searchValue,
+    } = props;
+
+    const dragValue = useMemo(() => ({
+        id: value.id,
+        title: value.title,
+        logoUrl: value.logoUrl,
+    }), [value]);
+
+    return (
+        <DraggableContent
+            name="stakeholder"
+            value={dragValue}
+            className={styles.item}
```

Comment:
Lets match the `className` with the name of component
```suggestion
            className={styles.stakeholder}
```